packages/babel-helpers/src/helpers/toSetter.js

```diff
@@ -0,0 +1,11 @@
+/* @minVersion 7.22.0 */
+
+export default function _toSetter(fn, thisArg) {
+  var args = [].slice.call(arguments, 2);
+  return Object.defineProperty({}, "_", {
+    set: function (v) {
+      args.push(v);
```

Comment:
The return value of `toSetter` does not seem to be idempotent, can we name it `toXxxSetter`?

Reply:
`toOnceSetter`, since it's meant to only be used once? Or I'd prefer to just add a comment if that's ok.

Reply:
Both are fine for me.